conda/meta.yaml.in

```diff
@@ -4,24 +4,27 @@ package:
 
 source:
   git_rev: @UVCDAT_BRANCH@
-  git_url: git://github.com/UV-CDAT/vcdat.git
+  git_url: git://github.com/CDAT/vcdat.git
 
 requirements:
   build:
   - python
   - setuptools
   - nodejs
   run:
-  - python =2
+  - python=2
```

Comment:
are you sure we do not need a space here?

Reply:
I added a space.